settings.gradle.kts

```diff
@@ -10,5 +10,6 @@ plugins {
     id("org.hypertrace.version-settings") version "0.2.0"
 }
 
-rootProject.name = "alerting"
```

Comment:
Will we have `hypertrace-alert-manager` /  `hypertrace-alerting-system` or `hypertrace-alert-evalutor` as root project name eventually?

Do you think, we should have the same name for our repo too like hypertrace-service / hypertrace-ingester ?